src/providers/theme/colorPalette.ts

```diff
@@ -0,0 +1,37 @@
+import { PaletteMode } from '@mui/material';
+import { amber, deepOrange, grey } from '@mui/material/colors';
+import { PaletteOptions } from '@mui/material/styles';
+
+// TODO GH-20 Implement application light palette
+const lightPalette: PaletteOptions = {
```

Comment:
Are there default colors for background? `background` is missing here in light theme